src/components/LenderProfile/LenderTeamsList.vue

```diff
@@ -17,13 +17,13 @@
 					:href="`/team/${team.teamPublicId}`"
 				>
 					<kv-material-icon
-						v-if="!team.image.url"
+						v-if="!team.image?.url"
```

Comment:
I thought optional chaining didn't work in a Vue 2 template and that we had to use a computed value. Is this working locally when you open the page?

Reply:
Oh nice call... As this is a loop I'll add a method to get the image url.